backend/app/utils/Whisper.scala

```diff
@@ -6,7 +6,7 @@ import java.nio.file.Path
 import scala.io.Source
 import scala.sys.process._
 
-case class TranscriptionResult(text: String, language: String)
+case class WhisperResult(text: String, language: String)
```

Comment:
renamed as we need to use 'TranscriptionResult' for the type of the combined output